src/EA.Weee.Web/Views/OrganisationRegistration/PartnershipDetails.cshtml

```diff
@@ -1,45 +1,45 @@
 ﻿@using EA.Weee.Core.DataStandards
-@using EA.Weee.Web.Areas.Admin.Controllers
 @using MvcNavigationHelpers
 @using EA.Weee.Web.Controllers
-@model EA.Weee.Web.Areas.Admin.ViewModels.AddOrganisation.Details.PartnershipDetailsViewModel
+@using EA.Weee.Web.RazorHelpers
+@model EA.Weee.Core.Organisations.PartnershipDetailsViewModel
 @{
-    ViewBag.Title = "Partnership details";
+    ViewBag.Title = "GOV.UK - Organisation details";
```

Comment:
is the "gov.uk - " needed here? Should we not use the ViewBag.Title in the header for these views for consistency.  Since we've done that for the other screens

Reply:
Good spot - it actually should be "Title" - GOV.UK, so we should add that to any of the existing screens when we see it